### PR TITLE
admin: admin show ddl jobs add tableName, schema name

### PR DESCRIPTION
**Pull request by ciscoxll**
Added field tableName and  schema name, ddl job history sql limit 20.

before shows results
```
admin show ddl jobs
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
| JOBS                                                                                                                                                                                               | STATE  |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
| ID:30, Type:create table, State:synced, SchemaState:public, SchemaID:27, TableID:29, RowCount:0, ArgLen:0, start time: 2018-02-28 11:01:15.834 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:28, Type:create schema, State:synced, SchemaState:public, SchemaID:27, TableID:0, RowCount:0, ArgLen:0, start time: 2018-02-28 10:54:16.384 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:26, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:25, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:57.501 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:24, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:23, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:57.388 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:22, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:21, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:57.276 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:20, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:19, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:57.162 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:18, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:17, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:57.05 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0   | synced |
| ID:16, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:15, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:56.935 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:14, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:13, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:56.824 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
| ID:12, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:11, RowCount:0, ArgLen:0, start time: 2018-02-26 11:31:56.712 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0  | synced |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
```

Now shows results
```
admin show ddl jobs
+--------+---------+------------------+--------------+--------------+-----------+----------+-----------+-----------------------------------+--------+
| JOB_ID | DB_NAME | TABLE_NAME       | JOB_TYPE     | SCHEMA_STATE | SCHEMA_ID | TABLE_ID | ROW_COUNT | START_TIME                        | STATE  |
+--------+---------+------------------+--------------+--------------+-----------+----------+-----------+-----------------------------------+--------+
|     30 | test    | base             | create table | public       |         1 |       29 |         0 | 2018-04-11 17:03:12.404 +0800 CST | synced |
|     28 | mysql   | stats_feedback   | create table | public       |         3 |       27 |         0 | 2018-04-11 16:58:16.909 +0800 CST | synced |
|     26 | mysql   | gc_delete_range  | create table | public       |         3 |       25 |         0 | 2018-04-11 16:58:16.796 +0800 CST | synced |
|     24 | mysql   | stats_buckets    | create table | public       |         3 |       23 |         0 | 2018-04-11 16:58:16.682 +0800 CST | synced |
|     22 | mysql   | stats_histograms | create table | public       |         3 |       21 |         0 | 2018-04-11 16:58:16.57 +0800 CST  | synced |
|     20 | mysql   | stats_meta       | create table | public       |         3 |       19 |         0 | 2018-04-11 16:58:16.46 +0800 CST  | synced |
|     18 | mysql   | help_topic       | create table | public       |         3 |       17 |         0 | 2018-04-11 16:58:16.346 +0800 CST | synced |
|     16 | mysql   | tidb             | create table | public       |         3 |       15 |         0 | 2018-04-11 16:58:16.235 +0800 CST | synced |
|     14 | mysql   | global_variables | create table | public       |         3 |       13 |         0 | 2018-04-11 16:58:16.125 +0800 CST | synced |
|     12 | mysql   | columns_priv     | create table | public       |         3 |       11 |         0 | 2018-04-11 16:58:16.011 +0800 CST | synced |
+--------+---------+------------------+--------------+--------------+-----------+----------+-----------+-----------------------------------+--------+
```

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/pingcap/tidb/util/ranger"
 	log "github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
+	"time"
```
Comment (jackysp) [inline executor/executor.go]: Move this line under line 20.

Comment (zhexuany) [reply executor/executor.go]: ditto

**Comment by zhexuany** [comment]
@ciscoxll  please address reviewer's comment. Expect Jack's comment, other changes look good to me.

**Comment by winkyao** [comment]
Please make the commit log clear next time, you can use git rebase to do this.

**Comment by zhexuany** [comment]
@jackysp PTAL

File: model/ddl.go [hunk 2]
```diff
@@ -107,12 +107,14 @@ func (h *HistoryInfo) Clean() {
...
-	ID       int64         `json:"id"`
-	Type     ActionType    `json:"type"`
-	SchemaID int64         `json:"schema_id"`
-	TableID  int64         `json:"table_id"`
-	State    JobState      `json:"state"`
-	Error    *terror.Error `json:"err"`
+	ID         int64         `json:"id"`
+	Type       ActionType    `json:"type"`
+	SchemaID   int64         `json:"schema_id"`
+	SchemaName string        `json:"schema_name"`
```
Comment (zimulala) [inline model/ddl.go]: Could we not add these fields？
Can we get the schema name and table name from `Query`？

File: executor/executor.go [hunk 3]
```diff
@@ -337,13 +339,47 @@ func (e *ShowDDLJobsExec) Next(ctx context.Context, chk *chunk.Chunk) error {
...
+		chk.AppendInt64(7, e.jobs[i].RowCount)
+		chk.AppendString(8, tsConvert2Time(e.jobs[i].StartTS).String())
+		chk.AppendString(9, e.jobs[i].State.String())
 	}
 	e.cursor += numCurBatch
 	return nil
 }
 
+// tsConvert2Time converts timestamp to time.
+func tsConvert2Time(ts uint64) time.Time {
```
Comment (zimulala) [inline executor/executor.go]: Could we use the `model. tsConvert2Time `? We can rename it to TSConvert2Time.

**Comment by shenli** [comment]
@zimulala @winkyao PTAL

**Comment by ciscoxll** [comment]
@winkyao @zimulala PTAL

**Comment by ciscoxll** [comment]
@winkyao  PTAL

**Comment by ciscoxll** [comment]
@winkyao PTAL

**Pull request merged**